tests/ui_tests/__init__.py

```diff
@@ -186,6 +195,31 @@ def write_fixtures_suggestion(
     )
 
 
+def update_fixtures_with_diff() -> int:
```

Comment:
could we use this to fix `test_emu_ui_record_multicore` somehow? (to recall: the issue is that the test records write over one another)

(although that would probably require more changes, i.e., saving a fixtures diff per runner and then finishing it up in the coordinator.)

Reply:
It really seems the issues with `test_emu_ui_record_multicore` are harder to solve.

We can define `test_emu_ui_record_multicore` = `test_emu_ui_multicore` + `test_emu_accept_fixtures` - I tried it in [c4d30db](https://github.com/trezor/trezor-firmware/pull/2744/commits/c4d30dbd1c5618f7dab668c584c819826c8dd230) and it works fine (we only need to catch the error code from the first command not to fail the whole "pipeline")

Reply:
looks fine, it's still technically possible for two writes to coincide but in practice it's super unlikely that two calls to `write()` with data this short would interleave at the kernel level

a small drawback is that this causes the write even if the tests fail, but eh :woman_shrugging: